core/services/ardupilot_manager/mavlink_proxy/AbstractRouter.py

```diff
@@ -65,7 +65,17 @@ def possible_interfaces() -> List[str]:
 
     @staticmethod
     def available_interfaces() -> List[Type["AbstractRouter"]]:
-        return list(filter(lambda subclass: subclass.is_ok, AbstractRouter.__subclasses__()))
+        logger.debug(f"Possible interfaces: {AbstractRouter.possible_interfaces()}")
+
+        def caller(subclass: Type["AbstractRouter"]) -> bool:
```

Comment:
can we use a more meaningfull name than "caller" ?
maybe `router_is_ok()`, `interface_is_ok()` or similar?